tests/ui/issues/issue-16151.rs

```diff
@@ -10,6 +10,7 @@ impl Drop for Fragment {
     fn drop(&mut self) {
         unsafe {
             DROP_COUNT += 1;
+            //~^ WARN creating a mutable reference to mutable static is discouraged [static_mut_refs]
```

Comment:
Since this test is unrelated, let's fix (or `allow(..)` and mark `FIXME`) this and other similar uses instead.